images.CI/macos/anka/Service.Helpers.psm1

```diff
@@ -150,11 +228,22 @@ function Install-SoftwareUpdate {
     param(
         [Parameter(Mandatory)]
         [ValidateNotNullOrEmpty()]
-        [string] $HostName
+        [string] $HostName,
+        [array] $listOfUpdates
     )
-
-    $command = "sudo /usr/sbin/softwareupdate --all --install --restart --verbose"
-    Invoke-SSHPassCommand -HostName $HostName -Command $command
+    $osVersion = [Environment]::OSVersion
+    $osVersionMajorMinor = $osVersion.Version.ToString(2)
+    if ($osVersion.Version.Major -eq "12") {
```

Comment:
could you please left a comment why we do so

Reply:
Done